docs/rules/max-classes-per-file/index.html

```diff
@@ -0,0 +1,56 @@
+---
```

Comment:
please remove the generated docs from this PR; we'll add it when we make a release that contains this new rule.